webapp/src/viewModel/boardTree.ts

```diff
@@ -381,21 +385,21 @@ class MutableBoardTree implements BoardTree {
                     const aValue = a.properties[sortPropertyId] || ''
                     const bValue = b.properties[sortPropertyId] || ''
                     let result = 0
-                    if (template.type === 'select') {
+                    if (template.type === 'select' || template.type === 'multiSelect') {
                         // Always put empty values at the bottom
-                        if (aValue && !bValue) {
+                        if (aValue.length > 0 && bValue.length <= 0) {
                             return -1
                         }
-                        if (bValue && !aValue) {
+                        if (bValue.length > 0 && aValue.length <= 0) {
                             return 1
                         }
-                        if (!aValue && !bValue) {
+                        if (aValue.length <= 0 && bValue.length <= 0) {
                             return this.titleOrCreatedOrder(a, b)
                         }
 
                         // Sort by the option order (not alphabetically by value)
-                        const aOrder = template.options.findIndex((o) => o.id === aValue)
-                        const bOrder = template.options.findIndex((o) => o.id === bValue)
+                        const aOrder = template.options.findIndex((o) => o.id === (Array.isArray(aValue) ? aValue[0] : aValue))
```

Comment:
We decided that we should try to order alphabetically.

Reply:
Done!

Reply:
I noticed ordering alphabetically makes the if multiselect || select block statement almost idnetical to the else statement so I merged them together to avoid repeatable code